src/app/containers/ATIAnalytics/beacon/index.js

```diff
@@ -2,37 +2,26 @@ import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
 import { buildATIEventTrackUrl } from '../atiUrl';
 
 export const sendEventBeacon = async ({
-  element,
-  component,
+  componentName,
   type,
-  label,
+  componentInfo,
   ...params
 }) => {
-  const impressionProps = {
+  const eventProps = {
     ...params,
-    element,
-    component,
+    componentName,
+    componentInfo,
+    type,
   };
+
   const viewProps = {
-    ...impressionProps,
-    type,
-    label,
+    ...eventProps,
+    type: 'view',
   };
   await Promise.all([
-    sendBeacon(buildATIEventTrackUrl(impressionProps)), // impression event
-    sendBeacon(buildATIEventTrackUrl(viewProps)), // actual event
+    sendBeacon(buildATIEventTrackUrl(eventProps)), // event, e.g. click
+    sendBeacon(buildATIEventTrackUrl(viewProps)), // background event
```

Comment:
See comment https://github.com/bbc/simorgh/pull/4672/files#r351782475. From my understanding of looking into this, I think the background event should be firing when the element loads(? maybe becomes visible?) rather than when the click-handler fires. This is to allow the click-per-view metric to be measured. I _think_ even the `type` prop should remain as `click` rather than being changed to `type`, based on confluence documentation.

Reply:
Should be done as part of #4819